package.json

```diff
@@ -3,14 +3,15 @@
   "version": "2.0.0",
   "main": "./src/customfunctions.js",
   "scripts": {
-    "start": "start npm run dev-server && start npm run watch && npm run sideload",
-    "start-web": "start npm run dev-server && start npm run watch",
+    "start": "concurrently \"npm run dev-server\" && concurrently \"npm run watch\" && npm run sideload",
```

Comment:
I tried these changes out locally on Windows and the dev-server kicks off when running the start command, but the watch and sideload don't

Reply:
fixed with 607f0b5